casepro/backend/rapidpro.py

```diff
@@ -335,7 +335,7 @@ def push_outgoing(self, org, outgoing, as_broadcast=False):
     def push_contact(self, org, normalised_urn):
         client = self._get_client(org)
         contact = client.get_contacts(urn=f"{normalised_urn}")
-        if not contact:
+        if not contact.all():
             contact = client.create_contact(urns=[normalised_urn])
         return contact.uuid
```

Comment:
since `contact` is a query cursor object this won't work. you need to use `.first()` but only if `create_contact` also returns a query cursor object